components/account/OnboardingModal.tsx

```diff
@@ -0,0 +1,204 @@
+import { BaseDotsamaWallet } from "lib/wallets/base-dotsama-wallet";
+import { PolkadotjsWallet } from "lib/wallets/polkadotjs-wallet";
+import { SubWallet } from "lib/wallets/subwallet";
+import { TalismanWallet } from "lib/wallets/talisman-wallet";
+import { observer } from "mobx-react";
+import Image from "next/image";
+import { useState } from "react";
+
+interface StepperProps {
+  steps: number;
+  currentStep: number;
+  onStepClick?: (step: number) => void;
+}
+
+const Stepper = ({ steps, currentStep, onStepClick }: StepperProps) => {
+  return (
+    <div className="flex gap-x-[18px]">
+      {Array(steps)
+        .fill(null)
+        .map((_, index) => (
+          <button
+            onClick={() => onStepClick(index)}
+            disabled={index === currentStep}
+            className={`rounded-full h-[7px] w-[7px] ${
+              index === currentStep ? "bg-black" : "bg-sky-600"
+            }`}
+          ></button>
+        ))}
+    </div>
+  );
+};
+
+interface TextSectionProps {
+  headerText: string;
+  bodyText: string;
+  leftButtonText?: string;
+  rightButtonText?: string;
+  onLeftButtonClick?: () => void;
+  onRightButtonClick?: () => void;
+}
+
+const TextSection = ({
+  headerText,
+  bodyText,
+  leftButtonText,
+  rightButtonText,
+  onLeftButtonClick,
+  onRightButtonClick,
+}: TextSectionProps) => {
+  return (
+    <>
+      <div className="font-bold text-ztg-22-120">{headerText}</div>
+      <div className="text-center mb-auto">{bodyText}</div>
+      <div className="flex justify-center  gap-x-[20px] w-full px-[20px] h-[56px] font-medium">
+        {leftButtonText && (
+          <button
+            className="rounded-[100px] border-2 border-pastel-blue w-full"
+            onClick={onLeftButtonClick}
+          >
+            {leftButtonText}
+          </button>
+        )}
+        {rightButtonText && (
+          <button
+            className="rounded-[100px] border-2 border-pastel-blue w-full"
+            onClick={onRightButtonClick}
+          >
+            {rightButtonText}
+          </button>
+        )}
+      </div>
+    </>
+  );
+};
+
+const walletsConfig = [
+  new TalismanWallet(),
+  new PolkadotjsWallet(),
+  new SubWallet(),
+];
+
+const WalletSelection = observer(() => {
+  const handleWalletSelect = async (wallet: BaseDotsamaWallet) => {
+    window.open(wallet.installUrl);
```

Comment:
Is there a way to detect by browser and direct accordingly? I don't think it's a big issue, but I did notice that on Mozilla it goes to the Chrome extension instead.

Reply:
Detect what by browser?